src/styles/main.scss

```diff
@@ -0,0 +1,61 @@
+@import './utils/variables';
+@import './utils/mixins';
+
+body {
+  margin: 0;
+  height: 100vh;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to use `min-height` instead of `height` to prevent content from spilling out of the viewport on smaller screens or when zooming in.